lib/database_transform/schema.rb

```diff
@@ -23,7 +23,25 @@ def self.transform_table(source_table, args = {}, &proc)
     transform.instance_eval(&proc) if proc
   end
 
+  delegate :around_job, :thread, to: :class
   class << self
+    # Store the block if a block is given, otherwise the stored block will be returned.
+    def around_job(&block)
```

Comment:
I would define these outside the self singleton class, to be consistent with `transform_table`